client/blocks/conversation-caterpillar/docs/fixtures.js

```diff
@@ -52,4 +52,95 @@ export const comments = [
 		},
 		date: '2016-04-18T15:23:00+00:00',
 	},
```

Comment:
I see your point, but I pressed on with the fixtures so I could always have posts with certain attributes, e.g. post with no avatar URL to make sure it's correctly omitted.